benchmarks/hash_table/static_map/contains_bench.cu

```diff
@@ -46,24 +47,22 @@ std::enable_if_t<(sizeof(Key) == sizeof(Value)), void> static_map_contains(
   key_generator gen;
   gen.generate(dist_from_state<Dist>(state), keys.begin(), keys.end());
 
-  auto pairs_begin = thrust::make_transform_iterator(
-    keys.begin(), [] __device__(auto i) { return cuco::pair_type<Key, Value>(i, {}); });
+  thrust::device_vector<pair_type> pairs(num_keys);
+  thrust::transform(keys.begin(), keys.end(), pairs.begin(), [] __device__(Key const& key) {
+    return pair_type(key, {});
+  });
 
   cuco::static_map<Key, Value> map{size, cuco::empty_key<Key>{-1}, cuco::empty_value<Value>{-1}};
-  map.insert(pairs_begin, pairs_begin + num_keys);
-  CUCO_CUDA_TRY(cudaStreamSynchronize(nullptr));
+  map.insert(pairs.begin(), pairs.end());
```

Comment:
So the `insert` internally thus we don't have to manually sync right after?

Reply:
Yep, it will run in the default stream so the kernel syncs wrt the host